functional_tests/code-save/log-code/log-code.yea

```diff
@@ -0,0 +1,25 @@
+id: 0.code-save.3-log_code
```

Comment:
If the entire sub-directory of `py` files is saved, should we also add a test for that here? i.e. using the `contains` op to check for `another-lib.py` or `fake-lib.py`

Reply:
the files arent saved as files. they are saved in the artifact.   yea currently only knows about artifacts as opaque objects right now.   name and type, so that is all we test.   When it is extended to allow inspection of manifest, then it should be added as you suggested.

Reply:
Great, makes sense.